clang/lib/CConv/RewriteUtils.cpp

```diff
@@ -792,6 +436,10 @@ class TypeExprRewriter
       : Context(C), Info(I) , Writer(R) {}
 
   bool VisitCompoundLiteralExpr(CompoundLiteralExpr *CLE) {
+    // When an compound literal was visited in constraint generation, a
+    // constraint variable for it was stored in program info.  There should be
+    // either zero or one of these.
+    CVarSet CVSingleton = Info.getPersistentConstraintVars(CLE, Context);
```

Comment:
But `CVSingelton` is not used after you call this method?

Reply:
Looks like I missed this during a merge.